bundles/org.openhab.binding.withings/src/main/java/org/openhab/binding/withings/internal/handler/WithingsBridgeHandler.java

```diff
@@ -0,0 +1,239 @@
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.withings.internal.handler;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import java.util.concurrent.*;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.jdt.annotation.Nullable;
+import org.eclipse.jetty.client.HttpClient;
+import org.openhab.binding.withings.internal.WithingsBindingConstants;
+import org.openhab.binding.withings.internal.api.WithingsDataModel;
+import org.openhab.binding.withings.internal.api.auth.AuthHandler;
+import org.openhab.binding.withings.internal.api.auth.WithingsAccessTokenResponse;
+import org.openhab.binding.withings.internal.api.device.DevicesHandler;
+import org.openhab.binding.withings.internal.api.device.DevicesResponse;
+import org.openhab.binding.withings.internal.config.WithingsBridgeConfiguration;
+import org.openhab.binding.withings.internal.service.AccessTokenInitializableService;
+import org.openhab.binding.withings.internal.service.person.Person;
+import org.openhab.binding.withings.internal.service.person.PersonHandler;
+import org.openhab.core.auth.client.oauth2.*;
+import org.openhab.core.config.core.Configuration;
+import org.openhab.core.thing.Bridge;
+import org.openhab.core.thing.ChannelUID;
+import org.openhab.core.thing.ThingStatus;
+import org.openhab.core.thing.ThingStatusDetail;
+import org.openhab.core.thing.binding.BaseBridgeHandler;
+import org.openhab.core.types.Command;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * @author Sven Strohschein - Initial contribution
+ */
+@NonNullByDefault
+public class WithingsBridgeHandler extends BaseBridgeHandler {
+
+    private final Logger logger = LoggerFactory.getLogger(WithingsBridgeHandler.class);
+
+    private final HttpClient httpClient;
+    private final AccessTokenInitializableService accessTokenService;
+    private final List<WithingsThingHandler> thingHandlers;
+    public WithingsBridgeConfiguration configuration = new WithingsBridgeConfiguration();
+    private @Nullable ScheduledFuture<?> tokenRefreshJob;
+    private @Nullable ScheduledFuture<?> refreshJob;
+    private Optional<WithingsDataModel> currentModel;
+
+    public WithingsBridgeHandler(Bridge bridge, AccessTokenInitializableService accessTokenService,
+            HttpClient httpClient) {
+        super(bridge);
+        this.accessTokenService = accessTokenService;
+        this.httpClient = httpClient;
+        this.thingHandlers = new ArrayList<>();
+        this.currentModel = Optional.empty();
+    }
+
+    @Override
+    public void initialize() {
+        logger.debug("Initializing Withings API bridge handler.");
```

Comment:
Can this message be replaced by using the debugger or by increasing the framework's log level? See point 4 https://www.openhab.org/docs/developer/guidelines.html#f-logging

Reply:
solved / just removed